[WIP] TELCODOCS-269: D/S Docs & RN: MPINSTALL-6, Support Baremetal IPI clusters on hosts that have different provisioning NIC names (#36494)
openshift/openshift-docs: modules/ipi-install-additional-install-config-parameters.adoc

ifeval::[{product-version} >= 4.4]
a| [[provisioningNetworkInterface]]`provisioningNetworkInterface` |  | The name of the network interface on control plane nodes connected to the

> provisioning network. Optional as of {product-title} 4.4 and later releases, due the `bootMacAddress` field calculating the IP address for Ironic to bind to.

Reviewer: I wonder would it be better to restructure this sentences, like the other instances below.
Maybe:
As of {product-title} 4.4 and later releases, this filed is now optional due to the `bootMacAddress` field calculating the IP address for Ironic to bind to.

Author: Excellent suggestion! I've updated that line.

> provisioning network. As of {product-title} 4.4 and later releases, this field is now optional due to the `bootMacAddress` field calculating the IP address for Ironic to bind to.

Reviewer 2: ```suggestion
provisioning network. In {product-title} 4.4 and later releases, this field is now optional due to the `bootMacAddress` field calculating the IP address for Ironic to bind to.
```

I would also specify the field you are talking about. It maybe helpful for translation purposes.

Author: Excellent point! I erroneously thought that it would be ok to skip the specific reference, as I defined the field in the prior sentence. I think you're absolutely right that it should be included, especially for translation purposes. Thank you so much for the review!
